tests/hpc/enable_in_zypper.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable HPC repo through yast
+# Maintainer: mgriessmeier <mgriessmeier@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use lockapi;
+
+sub run() {
+
+    select_console 'root-console';
+    # disable packagekitd
+    script_run 'systemctl mask packagekit.service';
+    script_run 'systemctl stop packagekit.service';
+
+    my $repo     = get_required_var('HPC_REPO');
+    my $reponame = "Devel_HPC_SLE12SP2";
```

Comment:
if we store URL to repo in Test Suite variable probably we can do the same for reponame ?